prerendering-state-privacy-questionnaire.md

```diff
@@ -0,0 +1,72 @@
+#### 2.1 What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
```

Comment:
Nit: adding a h1 would be good, so that people who arrive at this document know which API it's talking about without having to look at the URL bar.

Reply:
Done.